.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "gui/app/third_party/qnanopainter"]
```

Comment:
What will we do if we need to make a bugfix to this library?

Reply:
I guess at first need for a fix we can fork and change the target? I honestly don't have a strong opinion in favor of submodules or local copy. @jlebar @jkff I guess you will have to reach an agreement. Keep in mind that keeping QtCharts it will it as hard, if not harder, to fix something in the within the lib..

Reply:
I think I mildly prefer a submodule.  I'm fine either way.

I think forking it is the right thing to do if we need to make a change.  A nice property of making our changes in a proper fork is, we can send PRs upstream!